Formula/c/curl.rb

```diff
@@ -3,11 +3,11 @@ class Curl < Formula
   homepage "https://curl.se"
   # Don't forget to update both instances of the version in the GitHub mirror URL.
   # `url` goes below this comment when the `stable` block is removed.
-  url "https://curl.se/download/curl-8.12.0.tar.bz2"
-  mirror "https://github.com/curl/curl/releases/download/curl-8_12_0/curl-8.12.0.tar.bz2"
-  mirror "http://fresh-center.net/linux/www/curl-8.12.0.tar.bz2"
-  mirror "http://fresh-center.net/linux/www/legacy/curl-8.12.0.tar.bz2"
-  sha256 "5a85adbe401ed3b998ee1128524e9b045feb39577f3c336f6997e7a4afaafcd7"
+  url "https://curl.se/download/curl-8.12.1.tar.bz2"
+  mirror "https://github.com/curl/curl/releases/download/curl-8_12_0/curl-8.12.1.tar.bz2"
```

Comment:
this is the issue cc @fxcoudert
 
```suggestion
  mirror "https://github.com/curl/curl/releases/download/curl-8_12_1/curl-8.12.1.tar.bz2"
```